website/docusaurus.config.js

```diff
@@ -3,11 +3,17 @@ module.exports = {
 	tagline: 'Powerfully simple build configuration',
 	url: 'https://premake.github.io/',
 	baseUrl: '/',
+	scripts: [
+		'https://use.fontawesome.com/dd1c9cd9ff.js'
+	],
 	onBrokenLinks: 'throw',
 	onBrokenMarkdownLinks: 'throw',
-	favicon: 'img/premake-logo.png', // FIXME: make actual favicon.ico file
+	favicon: 'img/favicon.ico',
 	organizationName: 'premake',
 	projectName: 'premake.github.io',
+	customFields: {
+		latestVersion5: '5.0.0-alpha15'
+	},
```

Comment:
I'm afraid this field is not used anywhere. I guess you wanted to extract it in `download.js` page to replace this hardcoded constant?

```js
const LATEST_VERSION = '5.0.0-alpha15';
```

Reply:
Nice catch, thanks. I went back and forth on the best way to set this up; missed this in my review.